bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/CleanupLedgerManager.java

```diff
@@ -34,11 +34,14 @@
 import org.apache.bookkeeper.versioning.Version;
 import org.apache.bookkeeper.versioning.Versioned;
 import org.apache.zookeeper.AsyncCallback;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * A ledger manager that cleans up resources upon closing.
  */
 public class CleanupLedgerManager implements LedgerManager {
+    private static final Logger LOG = LoggerFactory.getLogger(CleanupLedgerManager.class);
```

Comment:
It looks like this project supports Lombok.  I'm wondering if we can use it.

Reply:
Sure